src/main/java/com/xxxx/example/Servlet04.java

```diff
@@ -16,17 +16,19 @@
 public class Servlet04 extends HttpServlet {
     @Override
     protected void service(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        req.setCharacterEncoding("UTF-8");
         resp.setCharacterEncoding("UTF-8");
-        Database myDatabase = new Database("<connection string goes here>", "DatingAppStaging");
-        List<Document> doc2 = myDatabase.find_latest_posts(10);
-         // user this to query for a user.
+        Database myDatabase = new Database("", "DatingAppStaging");
+        List<Document> doc2 = myDatabase.find_latest_posts(20);
+//          user this to query for a user.
         for(int i = 0; i < doc2.toArray().length; i++){
             Document doc = doc2.get(i);
-            resp.getWriter().append(doc.toJson() + "しょうwがないけどねkom锛捕啦即托いあねfkおうだよ");
+            resp.getWriter().append(doc.toJson() + "tbs010143fniwufwifnj+)4733&3uoghqgushvsjcvbjbke3bfb34uofuvhduvwb1=f");
```

Comment:
It seems like we are using this `tbs010143fniwufwifnj+)4733&3uoghqgushvsjcvbjbke3bfb34uofuvhduvwb1=f` as a list separator, for multiple DB records. Can we properly put this inside an array and stringify instead?

This will not work if someone decide to put the exact string inside their message

Reply:
Yes, I use this instead since currently, I did not find a better way to convert Document type to JSON type. I will actually receive a String at the frontend, and I try to split them. Seems like this method is so far a plausible one. Waiting for the backtend to give a better shot.